### PR TITLE
adds Open In Cloudshell to all READMEs with one depth

### PR DESCRIPTION
**Pull request by bshaffer**
#493

### CODE REVIEW

File: auth/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Google Auth PHP Sample Application
 
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googlecloudplatform/php-docs-samples&page=editor&working_dir=auth&open_in_editor=README.md
```
Comment (tmatsuo) [inline auth/README.md]: I tried the link. There are basically 4 panes; file list, editor, instruction pane on the right, the shell at the bottom. I have feedback:

1. The editor and the instruction pane both shows the same file, but with the different format. I feel it's redundant.
2. The README is not perfect for this use case, the step for cloning the repo is redundant
3. Cloudshell doesn't have composer

Can we

1. Create a specialized readme file for this purpose and show it?
2. Can we install composer by default in the cloudshell environment?

Comment (bshaffer) [reply auth/README.md]: 1. If we remove "open_in_editor", it displays just the rendered README as expected, which I think should suffice. I'll get to updating that.

2. Agreed that we need `composer` in the cloud shell environment, and that it is a requirement for this PR. I'll get started on this.

**Comment by bshaffer** [comment]
This PR is dependent on composer being added to CloudShell, and CloudShell being upgraded to PHP 7.0.

**Comment by bshaffer** [comment]
Composer has been added to Cloud Shell and PHP has been upgraded to 7.0!

**Pull request merged**